packages/web-components/src/styles/patterns/input.styles.ts

```diff
@@ -187,7 +187,7 @@ export const inputStateStyles: (
   :host(:not([disabled]):focus-within:not(:active))::after {
     content: '';
     position: absolute;
-    height: 100%;
+    height: calc(${focusStrokeWidth} * 1px) solid ${accentFillRest};
```

Comment:
I was just about to look at this treatment to try to remove the curls at the edges. It looks like it can actually be fixed to 1px since the line is the border. Just needs to be not 0.

```suggestion
    height: calc(${focusStrokeWidth} * 1px);
```